sale_medical_prescription/views/sale_order_view.xml

```diff
@@ -9,6 +9,15 @@
         <field name="model">sale.order</field>
         <field name='inherit_id' ref='sale.view_order_form' />
         <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_view_invoice']" position="before">
+                <button type="action" class="oe_stat_button" icon="fa-list"
+                        name="%(medical_prescription.medical_prescription_order_line_action)d"
```

Comment:
This is old syntax, you can just use the actual method name here instead of converting to an int. If you look at this with debug mode enabled, you'll notice that the name is actually the DB Id of the method, which is damned near impossible to trace back vs. the method name on the model